pkg/runtime/reconciler.go

```diff
@@ -1126,16 +1154,23 @@ func NewReconcilerWithClient(
 	metrics *ackmetrics.Metrics,
 	cache ackrtcache.Caches,
 ) acktypes.AWSResourceReconciler {
+	rtLog := log.WithName("ackrt")
+	resyncPeriod := getResyncPeriod(rmf, cfg)
+	rtLog.V(1).Info("Initiating reconciler",
```

Comment:
It makes more sense to me to move this log message inside of `getResyncPeriod()` (and just pass the `rtLog` into that method as an optional parameter)

Reply:
I disagree a little bit here. The thing is that moving this log statement to `getResyncPeriod` have few flows:
- Rarely (maybe never) seen logger as a parameter to functions, we usually use context, which we don't have here, and there is no need to.
- We will have redundant log statements (at least 4 times) and maybe with parameter declaration changes we can use a `defer` but that is just over-engineering a simple function.
- It makes more sense to log that "Initiating controller with an X resync value" than saying "Chose X resync value)
- The goal is to help users know what's their resource resync value. Yes, it might be valuable to explain to users why we chose a certain value, but in that case, that should apply to every corner case in the runtime library which is just a killer IMO.

Reply:
OK, since `getResyncPeriod()` is only ever called here, then this makes sense.